exercises/isogram/isogram.tc

```diff
@@ -0,0 +1,49 @@
+log←'./isogram.tc.log'
+⍎ ')COPY ',(⎕FIO 30),'/../../test.apl'
+
+test∆copy_relative 'isogram.apl'
+test∆clear_log log
+
+⍝ check test version
+  1⎕CR	(version)
+'1.2.0'
+
+⍝ ∇isIsogram
+
+⍝ empty string
+  1⎕CR	(isIsogram '')
```

Comment:
The parentheses are neither necessary nor idiomatic for APL. Functions in APL have "long right scope", which means that everything after a function's name is considered as its right argument.